src/main/cpp/Robot.cpp

```diff
@@ -27,25 +31,29 @@ void Robot::TeleopInit() {
 
 void Robot::TeleopPeriodic() {
 
-  double dx = ljoy.GetRawAxis(1);
-  double dy = ljoy.GetRawAxis(0);
-  double dtheta = rjoy.GetX();
-  dx = abs(dx) < 0.1 ? 0.0: dx; 
-  dy = abs(dy) < 0.1 ? 0.0: dy;
-  dtheta = abs(dtheta) < 0.05 ? 0.0: dtheta;
-
-  joy_val_to_mps(dx);
-  joy_val_to_mps(dy);
-  joy_rot_to_rps(dtheta);
-
+  //get joystick input
+  double vx = ljoy.GetRawAxis(1);
+  double vy = ljoy.GetRawAxis(0);
+  double vtheta = rjoy.GetX();
```

Comment:
What is ljoy and rjoy, and what is the difference between .GetRawAxis(1) and GetX()?

Reply:
They are basically the same, the raw axis function just allows the axis to be specified by parameter instead of a different function. I agree the use of redundant functions is unclear; I'll check which ljoy axis is which next time I'm in shop and I'll change this.